### ISSUE TITLE
connection module (fivem)

### ISSUE THREAD

**Issue by Mesrine67** [Azuriom/Azuriom#384]
### Description of your suggestion

- add the connection via CFX (similar txadmin
fivem no longer requires steam, as for the most commonly used fivem frameworks no longer requires steam
fivem uses the following identifiers, fivem, license, discord

**Comment by MrMicky-FR** [comment]
I agree it would be nice, but I can’t find any documentation or resources on how a website can implement CFX login.

**Comment by Mesrine67** [comment]
examine the txadmin code?
I'm going to do some research if I find anything that might help I'll post it here

**Comment by Mesrine67** [comment]
https://forum.cfx.re/t/question-implement-login-with-fivem-support/4828975

https://github.com/citizenfx/fivem/blob/master/ext/webadmin/server/Authentication/DiscourseAuthenticationHandler.cs#L217
https://github.com/citizenfx/fivem/blob/master/ext/webadmin/server/Authentication/FxClaimsPrincipal.cs

https://meta.discourse.org/t/user-api-keys-specification/48536

**Comment by Mesrine67** [comment]
If you need someone to write lua code, I can help a little. I am currently in training and would be happy to help
authentication via Rockstar Social Club possible ? 

https://github.com/pablo-1610/cfx-api

https://www.jsdelivr.com/package/npm/full-fivem-api

https://github.com/PichotM/api-tebex

**Comment by Mesrine67** [comment]
can you interact with the fivem rconlog? 
I can try to create a similar resource, with actions that we offer you
you can create a code that returns several identifiers, old-esx = steam, new-esx = license, qbcore = cid or citizenid, ndcore = discord. then compare the fivem identifier with the identifiers found, if the identifier matches with another this finds the right players


https://github.com/citizenfx/cfx-server-data/tree/master/resources/%5Bsystem%5D/rconlog

**Comment by Mesrine67** [comment]
https://forum.cfx.re/t/login-with-idms-fivem-net-own-client-secret-client-id/4936156

**Comment by Mesrine67** [comment]
https://github.com/TheIndra55/discourse-user-api-keys

**Comment by Mesrine67** [comment]
https://github.com/citizenfx/discourse-patreon

**Comment by Mesrine67** [comment]
https://github.com/citizenfx/txAdmin/blob/361fed4796d229c91b00407c2df45a3aa4c761f2/core/components/AdminVault/providers/CitizenFX.js

**Comment by Mesrine67** [comment]
https://forum.cfx.re/t/how-to-get-a-players-fivem-fourms-username-from-id/1992169

**Comment by Mesrine67** [comment]
https://blog.discourse.org/2018/06/understanding-discourse-trust-levels/

https://forum.cfx.re/t/releases-rules-and-faq/240725

**Comment by Mesrine67** [comment]
Docs cfx.re create API https://docs.fivem.net/natives/?_0xF5C6330C
possible create API link server to web ? create system login

**Comment by Mesrine67** [comment]
https://github.com/TheCoati/Cfx.re-Socialite
